concent_api/api_testing_common.py

```diff
@@ -47,6 +48,11 @@
 REQUESTOR_ETHEREUM_PRIVATE_KEY_FOR_EMPTY_ACCOUNT = b'\x17\xc0\xd9\xd5}\x82\xa4\xe16\xa0C\xf5f\xda\xc4+\xf5(Y\x1ch\x8c\xf2B\x15\xb3\xb5D!\x18.\x04'
 REQUESTOR_ETHEREUM_PUBLIC_KEY_FOR_EMPTY_ACCOUNT = b'1\xbf\x84\x18*\xa8\x85\xb0\xfap\xbd!)\xf1/{\x1b}Q\x92\xf0o\xa7\x9b\xa7\x0b\xbd\x88\xff\xe2A\xa5b\x94m2!\xd3#E\x07\xe5\xe3\xb4!\xf3\xb9\xbe#\x8bc\xfbM\xe1\xee\x91\x00\x13\x17\xf6>x\xb8\xfc'
 
+REQUEST_HEADERS = {
+    'Content-Type': 'application/octet-stream',
+    'Concent-Golem-Messages-Version': GOLEM_MESSAGES_VERSION
```

Comment:
This is not request header that contain `golem-messages` version. Actually this is response header. The correct header name is `X-Golem-Messages`. Please check blueprint: https://github.com/golemfactory/concent/issues/857